lib/rules/prefer-ember-test-helpers.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const { getImportIdentifier } = require('../utils/import');
+
+//-------------------------------------------------------------------------------------
+// Rule Definition
+//-------------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description:
+        'enforce import of @ember/test-helpers method to ensure native method on `window` is not incorrectly used',
+      category: 'Testing',
+      recommended: false,
+      url:
+        'https://github.com/ember-cli/eslint-plugin-ember/tree/master/docs/rules/prefer-ember-test-helpers.md',
+    },
+    schema: [],
+  },
+
+  create: (context) => {
```

Comment:
I realized that we should use our `emberUtils.isTestFile` util function to ensure this rule only runs on test files.

Reply:
Oh right! I had that initially and ended up forgetting about it 😅